blueprints/devops/goreleaser/ops2deb.yml

```diff
@@ -0,0 +1,18 @@
+name: goreleaser
+matrix:
+  architectures:
+    - amd64
```

Comment:
May I suggest adding the increasingly popular `arm64` architecture ?

Reply:
Of course! When available we always add it!